setup.cfg

```diff
@@ -5,12 +5,13 @@ less-zeroes=True
 [isort]
 profile=black
 src_paths=cekit,docs,support,tests
-skip=__init__.py,.tox
+skip=__init__.py,.tox,venv
 
 [flake8]
 max-line-length = 88
 select = C,E,F,W,B,B950
 extend-ignore = E203, E501, W503
+exclude = venv,.tox
```

Comment:
I am not convinced by the `venv` references for flake8 and isort. Its not guaranteed that users will be using `venv` and its often recommended to use `.venv` instead (See https://docs.python.org/3/library/venv.html ) - which is what I also use with https://github.com/MichaelAquilina/zsh-autoswitch-virtualenv/ though I host all my virtualenvs in `$HOME/.virtualenvs`.

Reply:
this venv was created by IDE, mine ones are kept on another place.
as you asked before I'll remove it so it can be handled on another PR.

Reply:
it should be ready now.